### ISSUE TITLE
possible to add keybindings to jump between hunks?

### ISSUE THREAD

**Issue by mmcnl** [kahole/vscode-magit#92]
Hello and thanks so much for an amazing extension! It has made my transition from spacemacs significantly more pleasant!

One question: is it possible to add key bindings, in the magit status and diff views, to quickly jump from one hunk to the next? For example, in spacemacs evil-magit, you can press `ctrl+j` to move to the next collapsible section; in the case below, repeatedly pressing `ctrl+j` will move between the lines marked with `==>`:

```
==> Unstaged changes (2)
==> modified   README.md
==> @@ -1,2 +1,5 @@
    +
    +// sample change 1
    +
==> modified   shell/dot_bash_setup
==> @@ -32,6 +32,8 @@ 
    +
    +// sample change 2
    +
```

Thanks for considering this!

**Comment by kahole** [comment]
Good suggestion!
I'll attempt to do it by traversing the view tree upon command. This seems like the simplest, and non-intrusive, solution to start with. Am also considering implementing a SymbolProvider for the magit language and use it to navigate, but seems a bit much...

**Comment by kahole** [comment]
Now available in version 0.5.7 in marketplace.

**Comment by mmcnl** [comment]
Thanks so much for the speedy response!

**Comment by mmcnl** [comment]
@kahole Oh, I just noticed a small detail. If it's a large diff and the next hunk is below (or above) the edge of the screen, when using `magit.move-next-entity` the cursor disappears until you make another movement (eg `j` or `k` or use) or press `esc` and then the viewport jumps to meet it.

**Comment by kahole** [comment]
should be fixed in version 0.5.8
